apps/condo/domains/meter/schema/Meter.test.js

```diff
@@ -847,7 +847,7 @@ describe('Meter', () => {
                 expect(meters).toHaveLength(0)
             })
 
-            test('resident: cannot read Meters in other unit in same property', async () => {
+            test('resident: can read Meters in other unit in same property if accountNumber is correct', async () => {
```

Comment:
Is there a test for case, where accountNumber is not correct?

Reply:
Done